packages/cli/src/decorators/controller.registry.ts

```diff
@@ -116,7 +116,11 @@ export class ControllerRegistry {
 				...(route.accessScope ? [this.createScopedMiddleware(route.accessScope)] : []),
 				...controllerMiddlewares,
 				...route.middlewares,
-				route.usesTemplates ? handler : send(handler),
+				route.usesTemplates
+					? async (req, res) => {
+							await handler(req, res);
```

Comment:
I see this function returns things, can we comment that we intentionally drop the return value here if that's intentional?

```suggestion
              // Intentionally drop return value
							await handler(req, res);
```